diff --git a/src/exports/vite/index.js b/src/exports/vite/index.js
index 95881be7aa864e4ca1e92292ddaaae4226c0097f..b03ffe510aefba0bcea2e7c20cfb82cca8b24976 100644
--- a/src/exports/vite/index.js
+++ b/src/exports/vite/index.js
@@ -932,7 +932,7 @@ async function kit({ svelte_config }) {
 								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
 								hoistTransitiveImports: false,
 								sourcemapIgnoreList,
-								inlineDynamicImports: !split
+								codeSplitting: split
 							},
 							preserveEntrySignatures: 'strict',
 							onwarn(warning, handler) {
